drafts/2019-10-15-openliberty-with-scala.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Calculating Mandelbrot Set using OpenLiberty microservices implemented in Scala"
+date:   2018-10-15 20:00:00 -0000
+categories: blog
+author_picture: https://avatars1.githubusercontent.com/u/50119017
+author_github: https://github.com/krzsam
+seo-title: Calculating Mandelbrot Set using OpenLiberty microservices implemented in Scala - OpenLiberty.io
+seo-description: An example application to calculate a Mandelbrot Set using Microprofile/Open Liberty microservices implemented in Scala. The goal is to demonstrate technical possibility of using OpenLiberty with Scala and possibility of deploying such solution as Kubernetes services.
+blog_description: "An example application to calculate a Mandelbrot Set using Microprofile/Open Liberty microservices implemented in Scala. The goal is to demonstrate technical possibility of using OpenLiberty with Scala and possibility of deploying such solution as Kubernetes services."
+---
+
+= Using Open Liberty in a Scala based application
+Krzysztof Samplawski <https://github.com/krzsam>
+
+This blog shows how Open Liberty microservices can be implemented using Scala language and deployed onto Kubernetes cluster.
+The main focus of the project was to explore technical feasibility of integration Open Liberty and Scala.
```

Comment:
"...explore the technical feasibility of integration between Open Liberty..."

Reply:
corrected